fluent-react/src/with_localization.js

```diff
@@ -25,7 +50,7 @@ export default function withLocalization(Inner) {
     render() {
       return createElement(
         Inner,
-        Object.assign({ getString: this.getString }, this.props)
+        Object.assign({ getString: (...args) => this.getString(...args) }, this.props)
```

Comment:
I think the linter complains about this line. Not that this change shouldn't be needed because `this.getString` is bound to `this` in line 9. That said, I'd much rather get rid of that `bind` and the whole `constructor` and fix the line length here :)

Reply:
Oh, that line does exist, I was on the wrong branch, sorry 🙈 I pushed a new commit, which also added a comment and removes the constructor.